build.sbt

```diff
@@ -632,7 +632,7 @@ lazy val test = project
     javaOptions in IntegrationTest += "-Xmx2G",
     testFrameworks += new TestFramework("scala.tools.partest.sbt.Framework"),
     testFrameworks -= new TestFramework("org.scalacheck.ScalaCheckFramework"),
-    testOptions in IntegrationTest += Tests.Argument("-Dpartest.java_opts=-Xmx1024M -Xms64M -XX:MaxPermSize=128M"),
+    testOptions in IntegrationTest += Tests.Argument(s"-Dpartest.java_opts=${(javaOptions in IntegrationTest).value.mkString(" ")}"),
```

Comment:
I like the idea of a large heap section.

Regarding the initial heap size, after this PR it would be 472M according to `java -Xmx2g -XX:+PrintFlagsFinal`. Should I move the `-Xms64m` setting to `javaOptions in IntegrationTest += "-Xmx2G",`?